geth/jail/jail.go

```diff
@@ -22,6 +22,8 @@ const (
 			return new Bignumber(val);
 		}
 	`
+	// ResponseCellCreated is returned when cell is successfully created and initialized.
+	ResponseCellCreated = `{ "result": "success" }`
```

Comment:
By default, Go returns compact JSON, so this should be `{"result":"success"}`.

Reply:
updated json to the compact notation